.travis.yml

```diff
@@ -13,20 +13,22 @@ addons:
     packages:
       - shellcheck
 
-install:
-  - pip install -U pip
+matrix:
+  fast_finish: true
+
+env:
+  - TEST_RUN="./tests/test-docker-compose.sh"
+  - TEST_RUN="./tests/test-kubernetes.sh"
```

Comment:
setting TEST_RUN twice?

Reply:
Yes, this causes Travis to run multiple jobs (one for docker-compose, and one for Kubernetes).

Reply:
ok.  Interesting that they went with the same syntax for that as for setting 2 vars. Not your problem though.